components/Form.js

```diff
@@ -0,0 +1,85 @@
+import styled from "styled-components";
+import { StyledButton } from "@/components/StyledButton";
+import { useRouter } from "next/router";
+
+export default function Form() {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    handleClick(event);
+  }
+
+  const router = useRouter();
+
+  const handleClick = (event) => {
+    event.preventDefault();
+    router.push("https://i.imgur.com/7KpCS0Y.jpg");
+  };
```

Comment:
you don't need this function.

The `router.push` can be part of your `handleSubmit`